jbt-it/neomdb
# #429 server implement automatic updating of last change and tracing

- Added a ChangeSubscriber that listens for updates and deletes
- Added UpdateDateColumn to lastChange of member

**How to test:**
- Run tests and check if successful
- Start client and server, change something in a department and a member and check in the database if the trace table has two new entries
- When the member is updated, the lastChange should also be updated in the database (correct date handling in the frontend is going to be implemented in #441)
